src/frontend/src/components/public/PIACollectionUseAndDisclosure/index.tsx

```diff
@@ -0,0 +1,192 @@
+import { useNavigate } from 'react-router-dom';
+import { isMPORole } from '../../../utils/helper.util';
+
+export interface TableColProps {
+  label: string;
+  filed: string;
+  value: string;
+  id: string;
+}
+
+const PIACollectionUseAndDisclosure = () => {
+  const navigate = useNavigate();
+  const [disclosure, setDisclosure] = useState('');
+  const [MPOCommentsDisclosure, setMPOCommentsDisclosure] = useState('');
+
+  const [rows, setRows] = useState([
+    [
+      { label: 'one', filed: 'one', value: '', id: 'one' },
+      { label: 'two', filed: 'two', value: '', id: 'two' },
+      { label: 'three', filed: 'three', value: '', id: 'three' },
+      { label: 'four', filed: 'four', value: '', id: 'four' },
+    ],
+    [
+      { label: 'one', filed: 'one', value: '', id: 'one' },
+      { label: 'two', filed: 'two', value: '', id: 'two' },
+      { label: 'three', filed: 'three', value: '', id: 'three' },
+      { label: 'four', filed: 'four', value: '', id: 'four' },
+    ],
+    [
+      { label: 'one', filed: 'one', value: '', id: 'one' },
+      { label: 'two', filed: 'two', value: '', id: 'two' },
+      { label: 'three', filed: 'three', value: '', id: 'three' },
+      { label: 'four', filed: 'four', value: '', id: 'four' },
+    ],
+    [
+      { label: 'one', filed: 'one', value: '', id: 'one' },
+      { label: 'two', filed: 'two', value: '', id: 'two' },
+      { label: 'three', filed: 'three', value: '', id: 'three' },
+      { label: 'four', filed: 'four', value: '', id: 'four' },
+    ],
+  ]);
+
+  const addRow = () => {
+    console.log(rows);
+    setRows([
+      ...rows,
+      [
+        { label: 'one', filed: 'one', value: '', id: 'one' },
+        { label: 'two', filed: 'two', value: '', id: 'two' },
+        { label: 'three', filed: 'three', value: '', id: 'three' },
+        { label: 'four', filed: 'four', value: '', id: 'four' },
+      ],
+    ]);
+  };
+  const isMPO = () => {
+    return isMPORole();
+  };
+  const removeRow = (index: number) => {
+    const newData = [...rows];
+    newData.splice(index, 1);
+    setRows(newData);
+  };
+  const handleBackClick = () => {
+    // 👇️ replace set to true
+
+    // TODO replace this with the correct  value in full pia scenario
+    navigate(-1);
+  };
+
+  const handleOnChange = (e: any, row: number, col: number) => {
+    const newData = rows.map((d, i) => {
+      if (i === row) {
+        d[col].value = e.target.value;
+      }
+
+      return d;
+    });
+    setRows(newData);
+  };
+
+  return (
+    <div className="bcgovPageContainer wrapper ">
+      <h1 className="results-header">{Messages.Headings.Title.en}</h1>
+      <span>{Messages.Headings.Subtitle.en}</span>
+      <h2 className="pt-3 pb-3">{Messages.WorkThroughDetails.Title.en}</h2>
+      <section className="card">
+        <div className="container">
+          <div className="row">
+            <div>
+              <table className="data-table__container" id="tab_logic">
```

Comment:
This list will be replace by my other pr( additional risk tab), I create a list component